sound/soc/sof/sof-client.c

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
+//
+// Copyright(c) 2020 Intel Corporation. All rights reserved.
+//
+// Author: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>
+//
+
+#include <linux/device.h>
+#include <linux/list.h>
+#include <linux/mutex.h>
+#include <linux/virtual_bus.h>
+#include "sof-client.h"
+
+static DEFINE_MUTEX(sof_client_mutex);
+
+static void sof_virtdev_release(struct virtbus_device *vdev)
+{
+	struct sof_client_dev *cdev = virtbus_dev_to_sof_client_dev(vdev);
+
+	kfree(cdev);
```

Comment:
`slab.h` Can be an incremental patch if you otherwise don't have to rework this set

Reply:
@lyakh for my education, what is the rationale for including the headers explicitly even if they are likely included from other headers?

Reply:
haha this is the result of a brain freeze. Will just remove the error